modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalazClientCodegen.java

```diff
@@ -76,8 +76,6 @@ public ScalazClientCodegen() {
         importMapping.remove("List");
         importMapping.remove("Set");
         importMapping.remove("Map");
-        
-        importMapping.put("ListBuffer", "scala.collection.mutable.ListBuffer");
```

Comment:
This is a ~badass~ breaking change and shouldn't be removed.

Edit: all your changes are badass, but I meant "breaking".